src/Controllers/LoginController.php

```diff
@@ -41,7 +41,7 @@ public function loginAction(RequestInterface $request)
             if ($this->session->get('role') === 'user') {
                 $response = new \Framework\Http\Response(Stream::createFromString(' '), []);
                 $response = $response->withStatus(301);
-                $response = $response->withHeader('Location', 'http://local.quiz.com/quiz/1');
+                $response = $response->withHeader('Location', 'http://local.quiz.com/quiz');
 
                 return $response;
```

Comment:
I will use the getRedirectPage from abstract controller, now it's not the goal of the current branch

Reply:
ok